cmake/onnxruntime_java.cmake

```diff
@@ -96,37 +96,68 @@ target_link_libraries(onnxruntime4j_jni PUBLIC onnxruntime)
 set(JAVA_PACKAGE_OUTPUT_DIR ${JAVA_OUTPUT_DIR}/build)
 file(MAKE_DIRECTORY ${JAVA_PACKAGE_OUTPUT_DIR})
 if (CMAKE_SYSTEM_NAME STREQUAL "Android")
-    set(ANDROID_PACKAGE_OUTPUT_DIR ${JAVA_PACKAGE_OUTPUT_DIR}/android)
-    file(MAKE_DIRECTORY ${ANDROID_PACKAGE_OUTPUT_DIR})
+  set(ANDROID_PACKAGE_OUTPUT_DIR ${JAVA_PACKAGE_OUTPUT_DIR}/android)
+  file(MAKE_DIRECTORY ${ANDROID_PACKAGE_OUTPUT_DIR})
 endif()
+
+# Set platform and ach for packaging
+if(CMAKE_SIZEOF_VOID_P EQUAL "8")
+    set(JNI_ARCH x64)
+else()
+   set(JNI_ARCH x86)
```

Comment:
The whitespace is off in this block, and until it's 32-bit compatible maybe it should error out if the Java api is requested for 32-bit architectures?